news/2020-04-07-cwc-2020-registrations-open.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "osu!catch World Cup 2020: Registrations now open!"
+date: 2020-04-07 09:00:00 +0000
+---
+
+Are you tired of being cooped up inside? Not sure what to do with your unspent energy? Do you feel the need to *dash*? The osu!catch World Cup returns for its' 9th installment to provide all of your exercise (and healthy eating!) needs — read on to find out how to register and more!
+
+![](/wiki/shared/news/banners/CWC2020.jpg)
+
+For more details, please refer to the **[osu!catch World Cup 2020 wiki article](/wiki/Tournaments/CWC/2020)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1046500)**.
```

Comment:
Are these going to be on the same line or separate line?

Reply:
They've been on the same line in previous World Cup announce posts, but I'm not particularly attached to this format if others think separating them would be better!